archinfo/arch.py

```diff
@@ -305,6 +311,13 @@ def library_search_path(self, pedantic=False):
             path = sum([[x + 'tls/${ARCH}/', x + 'tls/', x + '${ARCH}/', x] for x in path], [])
         return map(subfunc, path)
 
+    vex_support = True
+    unicorn_support = True
```

Comment:
both of these can be removed and inferred through whether vex_name and qemu_name are None